lib/addons/helm-addon/kubectl-provider.ts

```diff
@@ -113,15 +113,15 @@ export class KubectlProvider {
             data.replace(`{{${key}}}`, value);
         });
         return JSON.parse(data);
-    }
+    };
```

Comment:
did you use an automated tool setting in vscode to apply this consistently? can you share (or check in) the project config for this?

Reply:
I added the semi rule on .eslintrc [here](https://github.com/aws-quickstart/ssp-amazon-eks/blob/6ddb5703dadbc08b866721e84bfa56f18f622f3a/.eslintrc.js#L17), then used ESLint plugin to fix auto-fixable.